app/services/data/get-workload-points-configuration.js

```diff
@@ -5,9 +5,10 @@ const LocationPointsConfiguration = require('wmt-probation-rules').LocationPoint
 const DefaultNominalTargets = require('wmt-probation-rules').DefaultNominalTargets
 const DefaultContractedHours = require('wmt-probation-rules').DefaultContractedHours
 
-module.exports = function () {
+module.exports = function (isT2a) {
   return knex('workload_points')
     .orderBy('effective_from', 'desc')
```

Comment:
Need to update this to take the open record - i.e. effective_to is null. The ordering here is what's making the integration tests fail intermittently